src/components/field/index.js

```diff
@@ -330,6 +330,12 @@ export default class TextField extends PureComponent {
     }
   }
 
+  setBlurred() {
```

Comment:
This code is directly taken from `onBlur`, right? Can we remove the code from there and call `this.setBlurred()` instead?

Reply:
Good idea. Adapted it.